packages/modal/README.md

```diff
@@ -0,0 +1,104 @@
+# Modal
+
+![npm (scoped)](https://img.shields.io/npm/v/@leafygreen-ui/modal.svg)
+
+## Example
+
+```js
+import Modal, { ModalSize, useModalState } from '@leafygreen-ui/Menu';
+
+function ExampleComponent() {
+  const { active, setActive } = useModalState();
+
+  return (
+    <>
+      <button onClick={() => setActive(active => !active)}>Open Modal</button>
+      <Modal active={active} setActive={setActive}>
+        Modal Content goes here
+      </Modal>
+    </>
+  );
+}
+```
+
+**Output HTML**
+
+```html
+<button>Open Modal</button>
+<div>
+  <div class="leafygreen-ui-1wmxgfm"></div>
+  <div class="leafygreen-ui-2fvy7h" tabindex="-1">
+    <svg
+      width="24"
+      height="24"
+      role="img"
+      viewBox="0 0 16 16"
+      fill="#5D6C74"
+      class="leafygreen-ui-6hgda2"
+      data-dismiss="modal"
+      aria-hidden="true"
+      size="24"
+    >
+      <title>X Icon</title>
+      <path
+        d="M9.414 8l3.182 3.182-1.414 1.414L8 9.414l-3.182 3.182-1.414-1.414L6.586 8 3.404 4.818l1.414-1.414L8 6.586l3.182-3.182 1.414 1.414L9.414 8z"
+        fill="#5D6C74"
+        fill-rule="evenodd"
+      ></path>
+    </svg>
+    <div class="leafygreen-ui-j88j89">Modal Content goes here</div>
+  </div>
+</div>
+```
+
+## Properties
+
+### active
+
+**Type:** `boolean`
+
+**Default:** `false`
+
+Determines active state of `Modal` component
+
+### setActive
+
+**Type:** `function`
+
+**Default:** `(bool) => {}`
+
+Callback to set active state of Modal component. `setActive` optionally accepts a boolean value, which will determine the active state of `Modal` component.
```

Comment:
"optionally accepts a boolean value"? Is it optional?

Reply:
Thank you, good call